packages/flutter/test/material/search_anchor_test.dart

```diff
@@ -1188,19 +1188,19 @@ void main() {
       await tester.pumpWidget(buildSearchAnchor(platform));
```

Comment:
I cannot find a test using viewBuilder on the SearchAnchor.bar factory you added it to above.

Reply:
Added a test for this